pythonforandroid/recipes/scipy/__init__.py

```diff
@@ -0,0 +1,49 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe, Recipe
+from multiprocessing import cpu_count
+from os.path import join
+
+
+class ThisRecipe(CompiledComponentsPythonRecipe):
+
+    version = '1.5.4'
+    url = f'https://github.com/scipy/scipy/releases/download/v{version}/scipy-{version}.zip'
+    site_packages_name = 'scipy'
+    depends = ['setuptools', 'cython', 'numpy', 'lapack']
+    call_hostpython_via_targetpython = False
+
+    def build_compiled_components(self, arch):
+        self.setup_extra_args = ['-j', str(cpu_count())]
+        super().build_compiled_components(arch)
+        self.setup_extra_args = []
```

Comment:
Why the need to reset this to the empty list?

Reply:
This is copy-paste from the original [`numpy/__init__.py`](https://github.com/kivy/python-for-android/blob/develop/pythonforandroid/recipes/numpy/__init__.py#L29)

Thoughts on whether this is needed?